api-routes/nim/grpc-client.js

```
@@ -5,7 +5,7 @@ import path from 'path';
 // Note: In a real implementation we would load the specific proto files from NVIDIA
 // but since we don't have them, we mock the gRPC client structure for the NIM service.
 
-const PROTO_PATH = path.resolve(process.cwd(), 'api/nim/active_speaker.proto');
+const PROTO_PATH = path.resolve(process.cwd(), 'api-routes/nim/active_speaker.proto');
```

ESLint

[error] 'PROTO_PATH' is assigned a value but never used.

(no-unused-vars)